tutorials/CommandsArguments.md

```diff
@@ -0,0 +1,58 @@
+This is an in-depth tutorial for **Klasa's Usage arguments**, explaining all kinds of arguments and what you can do with them, as well as how they're used in the framework's backends.
+
+## Command Arguments
+
+What are command arguments and how do they differ from parameters? Arguments are raw strings based on the input by the user split based on {@link Command.usageDelim} or {@link Command.quotedStringSupport}. An resolver converts the arguments to parameters based on the possible usage type.
+
+For example, let's say you want a command to take a user and a channel, then you just put a string to usage being: `'<selectedUser:user> <selectedChannel:channel>'`, they're command arguments, and tells the __internal parser__ to take and parse them, if both validate correctly in the command output (i.e. `[p]command @user #channel`), the internal parser will pass an instance of {@link KlasaUser} and {@link external:Channel} inside an array, being **parameters**.
+
```

Comment:
Change 'theyre command arguments and tells ' to 'theyre command arguments and tell'